atlasmap-maven-plugin/src/test/java/io/atlasmap/maven/test/PaulsFieldActions.java

```diff
@@ -1,20 +1,14 @@
 package io.atlasmap.maven.test;
 
-import java.io.Serializable;
-
-import com.fasterxml.jackson.annotation.JsonPropertyDescription;
-
 import io.atlasmap.spi.AtlasFieldAction;
 import io.atlasmap.spi.AtlasActionProcessor;
-import io.atlasmap.v2.Action;
-import io.atlasmap.v2.AtlasActionProperty;
-import io.atlasmap.v2.FieldType;
+import io.atlasmap.v2.*;
 
 public class PaulsFieldActions implements AtlasFieldAction {
 
-    @AtlasActionProcessor
+    @AtlasActionProcessor(sourceType = FieldType.STRING)
```

Comment:
Since 2nd argument is String, this is not required. `sourceType` is used only to enforce to something else than auto detection.